app/build.gradle

```diff
@@ -38,35 +38,35 @@ android {
 }
 
 dependencies {
-    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    //kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$Versions.Tools.kotlin"
-    implementation 'androidx.appcompat:appcompat:1.1.0-alpha01'
-    implementation 'androidx.recyclerview:recyclerview:1.1.0-alpha01'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha3'
+
+    //android
+    implementation "androidx.appcompat:appcompat:$Versions.androidX"
+    implementation "androidx.recyclerview:recyclerview:$Versions.androidX"
+    implementation "androidx.constraintlayout:constraintlayout:$Versions.constraintLayout"
     implementation 'androidx.paging:paging-runtime:2.1.0-rc01'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/cd00bd156b5acad31bc778d24ad8b96a41fefaca/app/build.gradle#L48"></span>A newer version of androidx.paging:paging-runtime than 2.1.0-rc01 is available: 2.1.0</td>
    </tr>
  </tbody>
</table>